mbtools/models.py

```diff
@@ -167,7 +168,27 @@ def __init__(self, parent, location):
         self.parent = parent
         self.location = location
         self.etree = html.fromstring(self.parent.text)
-        # self.etree = html.fromstring(self.parent.text)
+
+    def prettify_html(self, html):
```

Comment:
> so the prettifying isn't necessarily tied to `replace_content_tag()`

I extracted the prettify function into utils class and called it when the file content is written.

Reply:
Any thoughts on whether we should still be using this at all given the following verbiage in their docs?:

> prettify() changes the meaning of an HTML document and should not be used to reformat one. The goal of prettify() is to help you visually understand the structure of the documents you work with.

Reply:
`Since it adds whitespace (in the form of newlines), ` Is there a case where this would change the output of the HTML? 🤔  Regardless if the docs warn against it we should not use it to reformat. I will remove the prettify code.